src/storage/store/column_chunk.cpp

```diff
@@ -550,6 +560,16 @@ class InternalIDColumnChunk final : public ColumnChunk {
         }
     }
 
+    void copyInt64VectorToBuffer(
+        ValueVector* vector, offset_t startPosInChunk, common::SelectionVector& selVector) {
+        for (auto i = 0u; i < selVector.selectedSize; i++) {
```

Comment:
An assert that it actually is an int64 vector might be helpful (I suppose it's in the function name, so it should be hard to mistake, but it's probably better to add those assertions whenever something isn't meant to work for all types).

Reply:
sg.